pkg/apis/agones/v1/gameserver.go

```diff
@@ -156,6 +156,8 @@ const (
 	// NodePodIP identifies an IP address from a pod.
 	NodePodIP corev1.NodeAddressType = "PodIP"
 
+	PassthroughPortAssignmentAnnotation = "autopilot.gke.io/container-passthrough-port-assignment"
```

Comment:
Use `agones.dev` rather than `autopilot.gke.io`. (In general, annotations are "owned" by the domain name - so it would be surprising to find the Agones project emitting a `gke.io` annotation.)

Reply:
Okok sounds good!